tests/test_utils.py

```diff
@@ -133,3 +160,36 @@ def test_derive_fingerprint_different_types_from_smiles():
     for i, fingerprint_type in enumerate(types):
         fingerprint = derive_fingerprint_from_smiles("[C+]#C[O-]", fingerprint_type, 16)
         assert numpy.all(fingerprint == expected_fingerprints[i]), "Expected different fingerprint."
+
+
+def test_missing_rdkit_module_error():
+    """Test if different functions return correct error when *rdkit* is not available"""
+    pytest.importorskip("rdkit")
+    expected_msg = "Conda package 'rdkit' is required for this functionality."
+
+    with mock.patch.dict(sys.modules, {"rdkit": None}):
```

Comment:
If rdkit is not installed this test will be skipped. Like when its run in `.github/workflows/CI_push_pypi.yml`  then I actually would like to run this test.

```suggestion
    try:
        from rdkit import Chem
        context = mock.patch.dict(sys.modules, {"rdkit": None})
    except ImportError:
        from contextlib import nullcontext
        context = nullcontext()

    expected_msg = "Conda package 'rdkit' is required for this functionality."
    with context:
```

Reply:
Added this, but pylint doesn't like "Import outside toplevel" and unused imports (Chem).